src/Illuminate/Auth/Access/Response.php

```diff
@@ -148,7 +148,11 @@ public function code()
     public function authorize()
     {
         if ($this->denied()) {
-            throw (new AuthorizationException($this->message(), $this->code()))
+            $message = $this->message();
```

Comment:
I realise this might not be the best way of going about it. Can someone more experienced offer some advice here?

Reply:
Would it be acceptable to use $_SERVER here?